docs/api-reference/attribute-manager.md

```diff
@@ -132,6 +134,8 @@ Parameters:
 * `buffers` (Object) - pre-allocated buffers
 * `props` (Object) - passed to updaters
 * `context` (Object) - Used as "this" context for updaters
+* `startIndex` (Number) - start index in the data object, if only a subset of data needs to be considered in the update
```

Comment:
We discussed supporting multiple ranges. Assuming that translates into multiple calls to this function.

Reply:
I am removing this parameter from `attributeManager.update` for now and implementing multiple range support in a follow up PR.